### PR TITLE
[NCP-1440] added texts for supporting testing a property

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

File: en_US.json [hunk 1]
```diff
@@ -2303,6 +2304,104 @@
...
+	"testsuites.button.save.and.create.case": "Save and Create Test Case",
+	"testsuites.button.search": "Search",
+	"testsuites.button.search.with.dataindex": "Search {dataIndex}",
+	"testsuites.button.yes.delete": "Yes, delete",
+	"testsuites.confirm.delete.multi": "Delete all({cntTestCase}) test cases?",
+	"testsuites.confirm.text.delete.test.case": "Delete the test case '{testCaseName}' ?",
+	"testsuites.confirm.text.delete.test.suite.name": "Delete the test suite '{testSuiteName}' ?",
+	"testsuites.confirm.text.delete.test.suite.name.more.one.test.cases": "Delete test suite '{testSuiteName}' and its {cntTestCases} test cases?",
+	"testsuites.confirm.text.delete.test.suite.name.only.one.test.cases": "Delete test suite '{testSuiteName}' and its {cntTestCases} test case?",
+	"testsuites.content.discard.case.create": "Discard the test case create?",
```
Comment (burtonquantil) [inline en_US.json:2324]: "create" should be "creation".

"create" is a verb only. We need the noun creation in this context.
This differs from "update" which can be used as both a verb and a noun.

File: en_US.json [hunk 2]
```diff
@@ -2303,6 +2304,104 @@
...
+	"testsuites.content.property.associated.info": "The property associated with this test suite is not currently deployed. Please deploy the property or select another property.",
+	"testsuites.content.success.save": "Test case has been successfully updated.",
+	"testsuites.content.successfully.updated": "Test suite has been successfully updated.",
+	"testsuites.empty.infomation": "Create test suites to test a property deployed to staging or production.",
+	"testsuites.error.title.create.test.suite": "Create Test Suite Error",
+	"testsuites.field.error.invalid.url": "Invalid URL",
+	"testsuites.field.error.required": "This field is required",
+	"testsuites.info.production": "To test on production, you must have an edge hostname and modify your DNS settings to direct traffic to the CDN Pro production platform. Click here for details.",
+	"testsuites.info.property.select": "Select a property deployed on staging or production.To test on production, you must have an edge hostname and modify your DNS settings to direct traffic to the CDN Pro production platform. Click here for details.",
```
Comment (burtonquantil) [inline en_US.json:2335]: There seems to be a missing space after the end of the first sentence.

File: en_US.json [hunk 3]
```diff
@@ -2303,6 +2304,104 @@
...
+	"testsuites.info.staging": "To test on staging, specify the staging server location by selecting a staging server location or entering an IP address. You can use a staging server in China only if your property hostname has an ICP Beian license.If left empty, CDN Pro will pick a staging server to test against.",
```
Comment (burtonquantil) [inline en_US.json:2336]: There should be a space before the last sentence which begins with "If"

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I made several small suggestions.

**Comment by jungho73** [comment]
I fixed some text by suggestion.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**